salt/_auth/kubernetes_rbac.py

```diff
@@ -177,7 +177,7 @@ def _load_kubeconfig(opts):
 
 
 @_check_auth_args
-def auth(username, token=None, **kwargs):
+def auth(username, token=None, **kwargs):  # pylint: disable=unused-argument
```

Comment:
I'm not sure, but I think there is some convention to describe "sink" arguments (ones that you will not use) to prefix them with an underscore.
How does pylint behave if you write instead:
```python
def auth(username, token=None, **_kwargs):
```
?

Reply:
For kwargs yes it's true we can do this. I will change this. 

Note that pylint point of view if you do a `def toto(_b=None)` for a `b` args that you do not use, pylint will report no error but then the signature of the function will be different